### PR TITLE
[Fix] Add missed checking for fastboot rendering

### PR DESCRIPTION
**Pull request by Rastopyr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-basic-dropdown",
-  "version": "0.12.0-beta.22",
```
Comment (cibernox) [inline package.json]: can you revert this line? It is incremented automatically what I run `ember release`

Comment (Rastopyr) [inline package.json]: Done :)

**Comment by Rastopyr** [comment]
Revert full commit, because have lint-error by jscs.
And, i think, we should link this PR with #105 issue.

**Comment by cibernox** [comment]
This week I won't be able to do do any OS, but I'll review all pending PRs over the weekend.

**Comment by cibernox** [comment]
Thanks

**Comment by Rastopyr** [comment]
np 👍

**Pull request merged**